lib/livebook_web/live/home_live/session_list_component.ex

```diff
@@ -171,6 +194,52 @@ defmodule LivebookWeb.HomeLive.SessionListComponent do
     """
   end
 
+  defp edit_sessions(assigns) do
+    ~H"""
+    <div class="mx-4 mr-2 text-gray-600 flex flex-row gap-1">
+      <%= if @editing_sessions? do %>
+        <.menu id="edit-sessions">
+          <:toggle>
+            <button class="w-28 button-base button-outlined-gray px-4 py-1 flex justify-between items-center">
+              <span>Actions</span>
+              <.remix_icon icon="arrow-down-s-line" class="text-lg leading-none align-middle ml-1" />
+            </button>
+          </:toggle>
+          <:content>
+            <a href="#" class="menu-item text-gray-600" phx-click="cancel_bulk_edit">
+              <.remix_icon icon="close-line" />
+              <span class="font-medium">Cancel</span>
+            </a>
+            <a href="#" class="menu-item text-gray-600" phx-click="select_all">
+              <.remix_icon icon="checkbox-multiple-line" />
+              <span class="font-medium">Select all</span>
+            </a>
+            <%= live_patch to: Routes.home_path(@socket, :edit_sessions, "disconnect"),
+              class: "menu-item text-gray-600 #{if length(@selected_session_ids) == 0, do: "menu-item--disabled"}",
```

Comment:
With length we need to go through the whole list, we can make a single coparison instead :)

```suggestion
              class: "menu-item text-gray-600 #{if @selected_session_ids == [], do: "menu-item--disabled"}",
```